App/Talent.App.WebApp/wwwroot/util/Talent-lib.js

```diff
@@ -96,3 +96,11 @@ TalentUtil.deepCopy = function (object) {
 TalentUtil.phoneRegExp = function () {
     return /^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/
 }
+
+TalentUtil.linkedInRegExp = function () {
+    return /^((http(s)?:\/\/)?(www\.)?linkedin\.([a-z]+)\/in\/([a-zA-Z0-9._-]+)\/?)?$/
+}
```

Comment:
This format is really confusing. I tried to add different linked in accounts and I can't save the form. It always show an invalid error message

Reply:
Thanks Kester, I have simplified the regex, hopefully now it will accept your linked-in account.  If it fails again, could you please send me the URL so I can better debug the regex?